pages/releases/4.3.html

```diff
@@ -0,0 +1,2313 @@
+				</div> <!-- #xr-performance-stability-improvements -->
+
+				<div id="xr-meta-xr-simulator-support" class="release-card span media-bottom">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-meta-xr-simulator-support">
+										Meta XR Simulator support
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										One complication of developing XR games is having to repeatedly test your
+										application on a headset, especially if your headset is a standalone VR
+										unit (read: not connected to a computer).
+									</p>
+									<p>
+										The Meta XR Simulator is a free tool to test your XR application outside
+										of the headset, but only works if you are targeting the Quest series models.
+										After downloading and installing it on your machine, you can now configure
+										it to work directly within your Godot editor. This feature is limited to
+										Windows and macOS because the simulator itself currently does not support
+										Linux.
+									</p>
+								</div>
+								<div class="c-link"
+									data-contributors="dsnopek,BastiaanOlij"
+									data-read-more="https://github.com/godotengine/godot/pull/85117">
+								</div>
+							</div>
+						</div>
+						<div class="release-card-media">
+							<img class="release-card-image position-top-left"
+								src="/storage/releases/4.3/images/xr-meta-xr-simulator.webp"
+								srcset="
+									/storage/releases/4.3/images/xr-meta-xr-simulator-2x.webp 2x
+								"
+								alt="Image of a Godot scene running in the Meta XR Simulator"
+								draggable="false"
+							/>
+						</div>
+					</div>
+				</div> <!-- #xr-meta-xr-simulator-support -->
+
+				<div id="xr-scene-discovery-spacial-anchors-on-meta-headsets" class="release-card media-top span-2">
+					<div class="release-card-container">
+						<div class="release-card-content">
+							<div class="release-card-content-container">
+								<h4 class="c-title">
+									<a href="#xr-scene-discovery-spacial-anchors-on-meta-headsets">
+										Meta headsets: scene discovery and anchors
+									</a>
+								</h4>
+								<div class="c-content">
+									<p>
+										Augmented reality requires you to have knowledge of the world around you
+										so you can interact with it. Meta provides you with this kind of information
+										via their scene discovery feature, which we are now supporting in Godot.
+										This way you can get a mesh of the world around you, or the position of
+										key elements in the room.
+									</p>
+									<p>
+										Spatial anchors let you save this data, so the system can remember locations
+										even after a restart. You can use them to for instance assign a permanent
+										location to a game object, or to anchor screens in one place.
```

Comment:
This isn't totally accurate. Spatial anchors don't allow you to save the scene data - they are about the developer adding _new_ data, and that data is saved.

So, basically, scene discovery data comes from the headset which tells us stuff about the real world space, and the developer has no control over that, we just read it. Spatial anchors are things the developer adds to be tracked in the real world space.

Maybe:

```suggestion
										Developers can add their own Spatial Anchors for the headset to track and remember,
										even after a restart. You can use them to for instance assign a permanent
										location to a game object, or to anchor virtual screens in one place.
```